java-sms-client/src/main/java/com/cumulocity/model/sms/LogMessageSanitizer.java

```diff
@@ -0,0 +1,38 @@
+package com.cumulocity.model.sms;
+
+public class LogMessageSanitizer {
```

Comment:
is it needed in this module?

Reply:
Can be moved to sms-client as it is used only there.
TODO